tests/core/persistent-journal-namespace/task.yaml

```diff
@@ -0,0 +1,61 @@
+summary: Test journal.persistent core config option is honored by journal namespaces
+
+# these systems do not support journal quota groups due to their old systemd versions.
+# requires systemd v245+
+systems:
+  - -ubuntu-core-16-*
+  - -ubuntu-core-18-*
+
+prepare: |
+  systemctl status snapd > before.txt
+  "$TESTSTOOLS"/snaps-state install-local test-snapd-journal-quota
+  snap set system experimental.quota-groups=true
+  tests.cleanup defer snap unset system experimental.quota-groups
+
+restore: |
+  # disable persistent journal
+  rm -rf /var/log/journal
+  systemctl kill --signal=SIGUSR1 systemd-journald
```

Comment:
nitpick: a comment explainign what this does (reloading the journal configuration, I guess?) would be helpful

Reply:
Adding a comment explaining it reloads the default journal